sql/core/src/test/scala/org/apache/spark/sql/execution/DataSourceScanExecRedactionSuite.scala

```diff
@@ -28,8 +28,10 @@ import org.apache.spark.sql.test.SharedSQLContext
  */
 class DataSourceScanExecRedactionSuite extends QueryTest with SharedSQLContext {
 
+  // TODO: create test suite for Data source V2 as well.
   override protected def sparkConf: SparkConf = super.sparkConf
     .set("spark.redaction.string.regex", "file:/[\\w_]+")
+    .set(SQLConf.USE_V1_SOURCE_READER_LIST.key, "parquet")
```

Comment:
That could benefit from the object with `val name = parquet`

Reply:
Too minor to use a variable for it.